src/khoj/processor/plaintext/plaintext_to_jsonl.py

```diff
@@ -0,0 +1,116 @@
+# Standard Packages
+import glob
+import logging
+from pathlib import Path
+from typing import List
+
+# Internal Packages
+from khoj.processor.text_to_jsonl import TextToJsonl
+from khoj.utils.helpers import get_absolute_path, timer
+from khoj.utils.jsonl import load_jsonl, compress_jsonl_data
+from khoj.utils.rawconfig import Entry
+
+
+logger = logging.getLogger(__name__)
+
+
+class PlaintextToJsonl(TextToJsonl):
+    # Define Functions
+    def process(self, previous_entries=[]):
+        # Extract required fields from config
+        input_jsonl_files, input_jsonl_filter, output_file = (
```

Comment:
These shouldn't be `input_jsonl_files`, `input_jsonl_filter`
```suggestion
        input_files, input_filter, output_file = (
```